help/export-your-organization.md

```diff
@@ -89,6 +98,37 @@ agreements and corporate policy for this type of export are in place. Note
 that many countries have laws that require employers to notify employees of
 their use of such an export.
 
+The output of this export is suitable to import into an on-premise
+installation of Zulip. It is not designed to be human-readable.
+
+## Compliance export
+
+{!owner-only.md!}
+
+This export is limited to paid Zulip Cloud Standard customers, though
```

Comment:
We generally link `Zulip Cloud Standard` to https://zulip.com/plans/

Reply:
This is copied verbatim from the full export without member consent paragraph above.  I'll link both.